chromium_src/components/affiliations/core/browser/affiliation_service_impl.cc

```diff
@@ -0,0 +1,27 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "components/affiliations/core/browser/affiliation_service_impl.h"
+
+#define AffiliationServiceImpl AffiliationServiceImpl_ChromiumImpl
+#include "src/components/affiliations/core/browser/affiliation_service_impl.cc"
+#undef AffiliationServiceImpl
+
+namespace affiliations {
+
+void AffiliationServiceImpl::PrefetchChangePasswordURL(
```

Comment:
I'm pretty sure we should be disabling this entire service cc @ShivanKaul

Reply:
I thought @mkarolin did this? https://bravesoftware.slack.com/archives/CU5UXM3EC/p1744745729794639

Reply:
He had said:

> unfortunately AffiliationService expected not to be null everywhere it's used .. so we may have to override (Pre)+fetch

Reply:
Can you open an issue for it?

Reply:
Created follow up https://github.com/brave/brave-browser/issues/45993